replications/internal/queue_management_test.go

```diff
@@ -417,20 +418,14 @@ func TestEnqueueData_WithMetrics(t *testing.T) {
 		require.Equal(t, i*len(data), int(totalBytesQueued.Counter.GetValue()))
 
 		currentBytesQueued := getPromMetric(t, "replications_queue_current_bytes_queued", reg)
-		// 8 bytes for an empty queue; 8 extra bytes for each byte slice appended to the queue
```

Comment:
Using `TotalBytes` allowed this test to be simplified - the metric doesn't include the footer bytes anymore, and we don't have to mess around the with the maximum disk size to get the change. Calling `SendWrite` processes the data in the queue and when that's done the remaining queue to process is 0.